galaxy_transform.py

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/python3
+
+import sys
+from ruamel.yaml import YAML
+
+filepath = "galaxy.yml"
+
+buf = open(filepath).read()
+
+yaml = YAML()
```

Comment:
try `yaml = YAML(typ="rt")`
That might help preserve quoting, indentation, etc.

Reply:
well, you need that, and this:
```python
yaml.default_flow_style = False
yaml.preserve_quotes = True
yaml.indent(mapping=2, sequence=4, offset=2)
```

Reply:
That gives the following:
```yaml
namespace: "mynamespace"
name: "myname"
version: "5.4.3"
description: "Ansible roles for linux system components management"

authors:
  - "System Roles Owners <systemroles@lists.fedorahosted.org>"

repository: "https://github.com/linux-system-roles/linux-system-roles.github.io/"
documentation: "https://github.com/linux-system-roles/linux-system-roles.github.io/blob/master/README.md"
homepage: "https://linux-system-roles.github.io"
issues: "https://github.com/linux-system-roles/linux-system-roles.github.io/issues"

readme: "README.md"
license:
  - GPL-2.0-or-later
  - GPL-3.0-only
  - GPL-3.0-or-later
  - LGPL-3.0-only
  - MIT
  - BSD-3-Clause

dependencies:

tags:
  - "system"
  - "collection"
```

Reply:
No luck so far...

My ruamel.yaml version is 0.16.12.

Reply:
Please note that it is not a critical issue. There were no errors nor even a warning in uploading to the test automation-hub with the following galaxy.yml. So, we don't have to worry about it, I guess. I verified the generated MANIFEST.json looks valid.

I'm going to approve this pr again.
```
namespace: redhat
name: rhel_system_roles
version: 1.0.10
description: Ansible roles for linux system components management

authors:
- System Roles Owners <systemroles@lists.fedorahosted.org>

repository: https://github.com/linux-system-roles/linux-system-roles.github.io/
documentation: https://github.com/linux-system-roles/linux-system-roles.github.io/blob/master/README.md
homepage: https://linux-system-roles.github.io
issues: https://github.com/linux-system-roles/linux-system-roles.github.io/issues

readme: README.md
license:
- GPL-2.0-or-later
- GPL-3.0-only
- GPL-3.0-or-later
- LGPL-3.0-only
- MIT
- BSD-3-Clause

dependencies:

tags:
- system
- collection
 ```

Reply:
Thank you, @richm! I missed your comments while testing with automation-hub. :p

I verified your proposed code generates the valid galaxy.yml. I'd like to include your changes in this pr. > @pcahyna

Reply:
I added new commits with the proposed changes.